### PR TITLE
Updating getting_started.md File

### PR DESCRIPTION
**Pull request by azibit**
Update some parameter placement to help avoid some of the few issues I encountered when I started.

### CODE REVIEW

File: _tutorials/getting_started.md [hunk 1]
```diff
@@ -114,7 +114,9 @@ Num weight bits = 18
 
 This diagnostic ouput shows that the number of bits from the hash function is 18 (more than enough for this example).
 
-Use `-b bits` to adjust the number of bits to be used from the hash function.
+Use `-b <Number of bits>` to adjust the number of bits to be used from the hash function. 
+
+So you have `vw -b <Number of bits> house_dataset` to run the code again
```
Comment (jackgerrits) [inline _tutorials/getting_started.md]:
```suggestion
For example: `vw -b 10 house_dataset`
```

File: _tutorials/getting_started.md [hunk 2]
```diff
@@ -114,7 +114,9 @@ Num weight bits = 18
 
 This diagnostic ouput shows that the number of bits from the hash function is 18 (more than enough for this example).
 
-Use `-b bits` to adjust the number of bits to be used from the hash function.
+Use `-b <Number of bits>` to adjust the number of bits to be used from the hash function.
```
Comment (jackgerrits) [inline _tutorials/getting_started.md]:
```suggestion
Use `-b <number of bits>` to adjust the number of bits to be used from the hash function. 
```

File: _tutorials/getting_started.md [hunk 3]
```diff
@@ -128,7 +130,9 @@ The default learning rate is `0.5` with current default update (`--normalized --
 
 If the data is noisy, you need a larger data-set or multiple passes to predict well. For massive data-sets, the learning rate decays towards `0` by default.
 
-Use  `-l rate` to adjust the learning rate up or down.
+Use  `-l new_learning_rate` to adjust the learning rate up or down.
+
+So you can try something like `vw -l 0.4` if you want the new learning rate to be 0.4. You can always choose the learning rate based on your project
```
Comment (jackgerrits) [inline _tutorials/getting_started.md]:
```suggestion
Use `-l <learning rate>` to adjust the learning rate up or down.

For example: `vw -l 0.4`
```

File: _tutorials/getting_started.md [hunk 4]
```diff
@@ -140,7 +144,9 @@ The following output shows the initial time for learning rate decay:
 initial_t = 0
 ```
 
->**Note:** Learning rates often decay over time, and this diagnostic output specifies the initial time. You can adjust with `--initial_t time`, although this is rarely necessary these days.
+>**Note:** Learning rates often decay over time, and this diagnostic output specifies the initial time. You can adjust with `--initial_t new_time`, although this is rarely necessary these days.
```
Comment (jackgerrits) [inline _tutorials/getting_started.md]:
```suggestion
>**Note:** Learning rates often decay over time, and this diagnostic output specifies the initial time. You can adjust with `--initial_t <time>`, although this is rarely necessary these days.
```

File: _tutorials/getting_started.md [hunk 5]
```diff
@@ -140,7 +144,9 @@ The following output shows the initial time for learning rate decay:
...
+
+So you can have something like `vw --initial_t 4` if you want the new initial time to be 4
```
Comment (jackgerrits) [inline _tutorials/getting_started.md]:
```suggestion
For example: `vw --initial_t 4`
```

File: _tutorials/getting_started.md [hunk 6]
```diff
@@ -170,10 +176,14 @@ For example:
 
 Use `--cache_file housing.cache` to override the default cache file name.
 
+So you have something like `vw --cache_file housing.cache house_dataset` if you want to use the cache and not the original file
```
Comment (jackgerrits) [inline _tutorials/getting_started.md]: This example is misleading because house_dataset is implicitly the -d value here. Additionally if housing.cache does not exist it will generate it. I think line 177 summarizes what goes on here well enough.

File: _tutorials/getting_started.md [hunk 7]
```diff
@@ -170,10 +176,14 @@ For example:
...
+
 The cache file is created the first time you use `-c`. If the cache exists and is newer than the dataset, that file is used by default.
 
 Use `-c` for multiple passes `--passes`, so Vowpal Wabbit caches the data in a faster format (passes > 1 should be much faster).  If you want to experiment with the same dataset over and over, it is highly recommended to pass `-c` every time you train.
 
+So we can have something like: ```vw -c --passes 50 house_dataset```
```
Comment (jackgerrits) [inline _tutorials/getting_started.md]:
```suggestion
For example: `vw -c --passes 50 -d house_dataset`
```

**Comment by jackgerrits** [comment]
@azibit, I left some comments with suggestions to improve wording to better match the tone of the existing tutorial.

File: _tutorials/getting_started.md (hunk 6 above)
Comment (azibit) [reply _tutorials/getting_started.md]: I agree

File: _tutorials/getting_started.md [hunk 8]
```diff
@@ -140,7 +144,9 @@ The following output shows the initial time for learning rate decay:
 initial_t = 0
 ```
 
->**Note:** Learning rates often decay over time, and this diagnostic output specifies the initial time. You can adjust with `--initial_t time`, although this is rarely necessary these days.
+>**Note:** Learning rates often decay over time, and this diagnostic output specifies the initial time. You can adjust with `--initial_t <time>`, although this is rarely necessary these days.
+
+For example: `vw --initial_t 4`
```
Comment (jackgerrits) [inline _tutorials/getting_started.md:149]: This should be in the quote
